0.convert/nbconverted/0.sqlite_manifest.py

```diff
@@ -0,0 +1,145 @@
+
+
+# In[3]:
+
+
+# Connect to Cell Painting gallery S3 session
+# Note: you must have already configured awscli with credentials
+session = boto3.Session()
+s3 = session.resource("s3")
+
+cell_painting_gallery = s3.Bucket(s3_bucket_name)
+
+
+# In[4]:
+
+
+# Use client to list source directories
+# Sources represent different sites from the JUMP consortium
+client = boto3.client("s3")
+
+top_level_jump_result = client.list_objects(
+    Bucket=s3_bucket_name,
+    Prefix=f"{jump_id}/",
+    Delimiter="/"
+)
+
+data_collection_sources = []
+for source_directory in top_level_jump_result.get("CommonPrefixes"):
+    data_collection_sources.append(source_directory.get("Prefix").split("/")[1])
+    
+data_collection_sources
+
+
+# In[5]:
+
+
+# Obtain batch names for each source
+source_batch_dictionary = {}
+for source in data_collection_sources:
+    prefix_string_per_source = f"{jump_id}/{source}/{backend_directory}"
+    
+    source_prefix_where_batches_exist_client = client.list_objects(
+        Bucket=s3_bucket_name,
+        Prefix=prefix_string_per_source,
+        Delimiter="/"
+    )
+    source_batch_dictionary[source] = []
+    for batch_directory in source_prefix_where_batches_exist_client.get("CommonPrefixes"):
+        source_batch_dictionary[source].append(batch_directory["Prefix"].split("/")[-2])
+        
+source_batch_dictionary
+
+
+# In[6]:
+
+
+# Get plate ids
+source_plate_dictionary = {}
+source_plate_info = []
+for source in source_batch_dictionary:
+    source_plate_dictionary[source] = {}
+    source_batches = source_batch_dictionary[source]
+    
+    for batch in source_batches:        
+        prefix_string_per_batch = f"{jump_id}/{source}/{backend_directory}{batch}/"
+
+        source_prefix_where_plates_exist_client = client.list_objects(
+            Bucket=s3_bucket_name,
+            Prefix=prefix_string_per_batch,
+            Delimiter="/"
+        )
+        source_plate_dictionary[source][batch] = []
+
+        for plate_directory in source_prefix_where_plates_exist_client.get("CommonPrefixes"):
+            plate = plate_directory["Prefix"].split("/")[-2]
+            source_plate_dictionary[source][batch].append(plate)
+            
+            # Build SQLite file
+            sqlite_file = f"s3://{s3_bucket_name}/{prefix_string_per_batch}{plate}/{plate}.sqlite"
+            source_plate_info.append([source, batch, plate, sqlite_file])
+
+
+# In[7]:
+
+
+# Compile JUMP data manifest
+jump_df = pd.DataFrame(source_plate_info, columns=["soure", "batch", "plate", "sqlite_file"])
+
+# Append perturbation type to this dataframe
+# Load metadata from https://github.com/jump-cellpainting/datasets
+perturbation_type_df = pd.read_csv(perturbation_type_url)
+perturbation_cols = ["Metadata_Source", "Metadata_Batch", "Metadata_Plate"]
+
+jump_df = (
+    jump_df.merge(
+        perturbation_type_df,
+        left_on=["soure", "batch", "plate"],
```

Comment:
Double checking: should this be "source" instead?
```suggestion
        left_on=["source", "batch", "plate"],
```

Reply:
yep